test/unit/adapters/http.js

```diff
@@ -48,46 +46,6 @@ function toleranceRange(positive, negative) {
 
 var noop = ()=> {};
 
-const LOCAL_SERVER_URL = 'http://localhost:4444';
-
-function startHTTPServer(options) {
```

Comment:
this is removed and consolidated into a commons file so that it can be used to test the fetch adapter without duplicating code